extensions/send-to-flomo/src/send-to-flomo.tsx

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+import { useState } from "react";
+import {
+  getPreferenceValues,
+  ActionPanel,
+  Form,
+  FormValue,
+  SubmitFormAction,
+  showToast,
+  ToastStyle,
+} from "@raycast/api";
+
+interface Preferences {
+  api: string;
+  defaultTag: string;
+}
+
+export default function Command() {
+  const preferences: Preferences = getPreferenceValues();
+  const [text, setText] = useState<string>();
+
+  async function handleSubmit(values: Record<string, FormValue>) {
+    let content = values.memo;
+
+    if (preferences.defaultTag) {
+      content += ` ${preferences.defaultTag}`;
+    }
+
+    const response = await axios.post(preferences.api, { content });
+    if (response.data?.code === 0) {
+      showToast(ToastStyle.Success, "Success");
+      setText('');
+    } else {
+      showToast(ToastStyle.Failure, "Failed");
+    }
```

Comment:
```suggestion
if(content !== "") {

      if (preferences.defaultTag) {
        content += ` ${preferences.defaultTag}`;
      }

      showToast(ToastStyle.Animated, "Sending")

      const response = await axios.post(preferences.api, { content });

      if (response.data?.code === 0) {
        showToast(ToastStyle.Success, "Success", "Successfully send MEMO");
        setText('');
      } else {
        showToast(ToastStyle.Failure, "Failed", "Check that API MEMO URL is correct");
      }
    }
    else {
      showToast(ToastStyle.Failure, "Failed", "MEMO can't be left empty");
    }
```

Reply:
Added a toast spinner to indicate that it's being processed, it takes up to a few seconds to send, and this way we improve the UX. Also added an error if nothing is written.